sale_order_warehouse_location/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2024 Alberto Martínez <alberto.martinez@sygel.es>
```

Comment:
Thanks for the credits

Reply:
You deserve it. An __init__ file is quite complex to write, __innit_?